src/helpers/constants/ios.ts

```diff
@@ -1,6 +1,6 @@
 export const LOCAL_PATH_TO_CIO_NSE_FILES = `node_modules/customerio-expo-plugin/src/helpers/native-files/ios`;
 export const IOS_DEPLOYMENT_TARGET = '13.0';
-export const CIO_SDK_VERSION = '1.2.6';
+export const CIO_SDK_VERSION = '2.0.0-alpha.1';
```

Comment:
I am updating the SDK version so that I can test it with recent versions of `customerio-reactnative`

Reply:
@levibostian Do you think it's a good idea to leave this SDK version and ship it as is?

Reply:
Segun, is there a way to auto pull in the latest version of iOS SDK / CocoaPods ? Same for react native package version

Reply:
```suggestion
export const CIO_SDK_VERSION = '~> 2.0.0-alpha.1';
```

I like the idea of specifying the true syntax of the version string here in this constant. Instead of just the SDK version number, include the `~>` here, too. 

That does mean that we will need to update code like this in the code base: 

```
// before
export const CIO_PODFILE_SNIPPET = ` pod 'CustomerIO/MessagingPushAPN', '~> ${CIO_SDK_VERSION}'`;

// after
export const CIO_PODFILE_SNIPPET = ` pod 'CustomerIO/MessagingPushAPN', '${CIO_SDK_VERSION}'`;
```

Reply:
Aman, When you say, 

> Segun, is there a way to auto pull in the latest version of iOS SDK / CocoaPods ? Same for react native package version

Are you saying the latest *minor* version of the iOS and RN SDKs? My comments in this PR thus far do that. 

When I say minor version, I am saying, `>= 1.0, <2.0` so the expo plugin doesn't automatically upgrade from v1 to v2 of an SDK. The expo plugin will install v1.3 automatically, for example, but will not install v2.0 automatically which is what we want.